files/en-us/web/accessibility/aria/how_to_file_aria-related_bugs/index.html

```diff
@@ -37,7 +37,7 @@
   <tr>
    <td><a class="external" href="http://www.nvda-project.org/">Non Visual Desktop Access (NVDA)</a></td>
    <td><a class="external" href="http://www.nvda-project.org/wiki/ReportingIssues">File NVDA bugs</a></td>
-   <td><a href="/en/Accessibility/JAWS_Issues_with_Firefox" title="JAWS Issues with Firefox">Discuss NVDA issues</a></td>
+   <td><p>Discuss NVDA issues</p></td>
```

Comment:
```suggestion
   <td>Discuss NVDA issues</td>
```

We shouldn't need a paragraph element inside of a table cell.

Reply:
@ericwbailey There are a few in this table, I believe it was used for alignment purposes. Should I remove them all (it was probably a hack to get them to align to the top instead of being vertically centered)?

Reply:
That'd be great, thank you! If there's visual alignment issues it should be handled in the CSS.